### DISCUSSION TITLE
How to Integrate Umami Analytics in Nextjs with Planetscale

### DISCUSSION THREAD

**Discussion by giscus** [Evavic44/victoreke.com#96] [Announcements]
# How to Integrate Umami Analytics in Nextjs with Planetscale

Learn how to integrate Umami analytics into your nextjs application with PlanetScale and deploy on Vercel by following these steps

https://victoreke.com/blog/how-to-integrate-umami-analytics-in-nextjs-with-planetscale

**Comment by haghest** [comment]
with that large amount visitor and monthly event, are still using planetscale free tier version, or did they charge u already?

**Reply by haghest** [reply]
asking based on your thumbnail pic

**Reply by Evavic44** [reply]
Unfortunately PlanetScale no longer offers a free tier plan so I've longed moved this to [Umami cloud](https://umami.is/docs/cloud), which is hosted directly by Umami, ironically :)
